docs/source/bricks.rst

```diff
@@ -750,3 +750,26 @@ files to an S3 bucket.
 
   upload_staged_files()
 
+``stage_for_datasaur``
+--------------------------
+Formats a list of Text elements as input to Datasaur.
+
+For example, if input looks like:
```

Comment:
Make sure this is formatted like the docs for the other bricks. The other ones have:

```
Examples:
```

followed by the example code.